.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,19 @@
+# How to Contribute
+
+We welcome contributions from the community. Here are a few ways you can help us improve.
+
+## Open an Issue
+
+If you see something you'd like changed, but aren't sure how to change it, submit an issue describing what you'd like to see.
+
+## Submit a Pull Request
+
+If you feel like getting your hands dirty, feel free to make the change yourself. Here's how:
+
+1. Fork the repo on Github, and then clone it locally.
+2. Create a branch named appropriately for the change you are going to make.
+3. Make your code change.
+4. Push your code change up to your forked repo.
+5. Open a Pull Request to merge your changes to this repo. The comment box will be filled in automatically via a template.
+
+See [Using Pull Requests](https://help.github.com/articles/using-pull-requests/) got more information on how to use GitHub PRs.
```

Comment:
do you want to use our git-flow wiki for this? https://github.com/redhat-cop/git-flow/wiki